pom.xml

```diff
@@ -63,6 +63,10 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>biz.aQute.bnd</groupId>
```

Comment:
Please remove this unrelated change

Reply:
Is that a problem?

Reply:
Removed it again. If it is in parent we do not need it here.